README.md

```diff
@@ -29,6 +29,7 @@ deploying the operator assumes:
 * `make` targets are executed from the `openshift/cluster-logging-operator` root directory
 * various other commands such as `imagebuilder` and `operator-sdk` - it is suggested
   that you use the logging `sdk_setup.sh` script at https://raw.githubusercontent.com/openshift/origin-aggregated-logging/master/hack/sdk_setup.sh
+* Ensure you have `docker` package installed and `docker` daemon up and running on the workstation you are running these commands.
```

Comment:
This should probably just mention container runtime as CRIO replaces docker in 4.0 as far as I know.  @richm thoughts on how to properly word this?

Reply:
I think container runtime would be appropriate. I will make that change and would readd it.

Reply:
well - we currently use docker `docker push` to push images to the remote repo running in the cluster - is there another way to do that without docker?  `crictl` or `skopeo`?

Reply:
I'm working on a way to do that in another way with buildconfigs - but I think many devs will want to build images locally and push to the remote registry in the cluster

Reply:
@richm yes docker push is one which I could see being used for the image so maybe, for now, we could mention these updates on the doc so that users testing the repo would not find it difficult setting up the cluster-logging-operator.

Reply:
@jatanmalde for the typical QE use case, you won't need to build and push the images, so perhaps just a note that if you use `REMOTE_REGISTRY=true` you will need to have `docker` available

Reply:
@richm I added the changes now, could check if that would be okay?